src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -84,13 +85,15 @@ const BaseChatItem = ({item, index, type}: BaseChatItemComponentProps) => {
         data={item?.rawJson}
       />
     );
+  }
 
   return (
     <ChatItemTargetText
       key={index}
       avatar={handleAvatar()}
       isContinuous={item?.isContinuous ?? false}
       message={item?.message}
+      attachments={item?.attachmentJson}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the `attachments` prop is introduced to the `ChatItemTargetText` component. Again, ensure that this component can handle this new prop and that `item?.attachmentJson` is always in the expected format.

```diff
   return (
     <ChatItemTargetText
       key={index}
       avatar={handleAvatar()}
       isContinuous={item?.isContinuous ?? false}
       message={item?.message}
+      attachments={item?.attachmentJson}
       time={item?.updatedAt}
       username={handleUserName(item)}
       chatType={type ?? ANONYMOUS}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->